### DISCUSSION TITLE
How to download folder without zipping

### DISCUSSION THREAD

**Discussion by leugi101** [rejetto/hfs#826] [Q&A]
For some reason, when I add a folder to the shared files, and I download the folder, it's always saved a zip file. This makes it take twice as long to do anything because I have to extract the folder every time. I don't know if I am doing things wrong but I haven't been able to find a way around this. Any help appreciated.

**Comment by rejetto** [comment]
saving an actual folder is technically possible only with Chrome and Edge, and not with Firefox or Safari.
For this limited support I haven't worked on it, yet, and zipping is your only possibility.
Anyway, I will consider it.

**Reply by leugi101** [reply]
Thank you for the response.
